diagnostic.md

```diff
@@ -6,21 +6,23 @@ Record your responses inside the fenced code blocks below each question.
     and what are the main task(s) you perform inside an Ember Route?
 
     ```md
-    <!-- your response here -->
+    Router: Create the paths (i.e. 'routes') that will be used to manipulate data throughout the application. These routes are in the form of a URL path.
+
+    Route: Used to direct information throughout the application. Routes primarily consist of directions for the information to flow through.
     ```
 
 1.  What is the command to generate a route named `boston` nested under
     `campus`?
 
     ```md
-    <!-- your response here -->
+    'ember generate route campus/boston'
     ```
 
 1.  Suppose you have a nested route at the URL `/campus/boston`. How would you
     use the `link-to` helper to generate an appropriate link?
 
     ```md
-    <!-- your response here -->
+    {{#link-to 'boston'}} Here's the link to Boston! {{/link-to}}
```

Comment:
Because it's a nested route, it needs to be:
 `{{#link-to 'campus.boston'}} Here's the link to Boston! {{/link-to}}`